src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -212,13 +212,11 @@ object ZincRunner extends WorkerMain[ZincRunnerWorkerConfig] {
         .withSources(sources.map(source => PlainVirtualFile(source.toAbsolutePath().normalize())).toArray)
         .withClasspath((classesOutputDir +: deps.map(_.classpath)).map(path => PlainVirtualFile(path)).toArray)
         .withClassesDirectory(classesOutputDir)
-        .withJavacOptions(workRequest.javaCompilerOptions.toArray)
+        .withJavacOptions(workRequest.javaCompilerOptions)
         .withScalacOptions(
-          (
-            workRequest.plugins.map(p => s"-Xplugin:$p") ++
-              workRequest.compilerOptions ++
-              workRequest.compilerOptionsReferencingPaths
-          ).toArray,
+          workRequest.plugins.map(p => s"-Xplugin:$p").toArray ++
```

Comment:
Is there any benefit in doing `workRequest.plugins.view.map(p => s"-Xplugin:$p").toArray` to avoid the construction of intermediate results?

Reply:
If so, I kind of wonder if there would be a benefit to using views in other places as well. I doubt it would be a huge win, but probably something.

Reply:
Yup, you're right; that would be more performant. I wasn't able to find any places in `CommonArguments.scala` where `.view` could be effectively utilized, but I found a few other places in `ZincRunner`. I'll update them.